src/pages/hotel.js

```diff
@@ -3,155 +3,138 @@ import Layout from '@components/layout'
-              <TableRow>
-                <TableCell>Room Type</TableCell>
-                <TableCell>Price Per Night<sup>†</sup></TableCell>
-              </TableRow>
-            </TableHead>
-            <TableBody>
-              {rows.map((row, index) => (
-                <TableRow key={row.roomType}>
-                  <TableCell component="th" scope="row">
-                    {row.roomType}
-                  </TableCell>
-                  <TableCell>{row.price}</TableCell>
+        <CenteredBox>
+          <RoomRateTable>
+            <Table aria-label="simple table">
+              <TableHead style={{ textTransform: 'uppercase' }}>
+                <TableRow>
+                  <TableCell>Room Type</TableCell>
+                  <TableCell>Price Per Night<sup>†</sup></TableCell>
                 </TableRow>
-              ))}
-            </TableBody>
-          </Table>
-        </RoomRateTable>
-        
-        <div>
-          <sup>†</sup>Taxes and fees not included in price.
-        </div>
-        <Button variant="contained" className="cta" href='https://www.sonesta.com/sonesta-hotels-resorts/ca/redondo-beach/sonesta-redondo-beach-marina?isGroupCode=true&groupCode=G103025IDOL&checkin=2025-10-30&checkout=2025-11-02'>
-          Book online
-        </Button>
-        <div>If you prefer to book by phone, you can call the hotel toll-free at 310-318-8888 and reference the group code "G103025IDOL".</div>
-      </CenteredBox>
-              }
-      <CenteredBox>
-        <ResponsiveMap>
-          <iframe title="Hotel map" maxwidth="600" height="450" style={{ border: 0}} loading="lazy" allowFullScreen src="https://www.google.com/maps/embed/v1/place?q=place_id:ChIJn2IUCKC0woARmZBXFBOsAz0&key=AIzaSyDk8pOuoUbrYKocNBn6QzLe9j-bUAx7GvA"></iframe>
-        </ResponsiveMap>
+              </TableHead>
+              <TableBody>
+                {roomRates.map(room => (
+                  <TableRow key={room.label}>
+                    <TableCell component="th" scope="row">{room.label}</TableCell>
+                    <TableCell>{"$" + room.price}</TableCell>
+                  </TableRow>
+                ))}
+              </TableBody>
+            </Table>
+          </RoomRateTable>
 
-        <h2>How to Get Here</h2> 
-      </CenteredBox>
-      <h3>Bus</h3>
-        
-      <p>The hotel is accessible via Beach City Transit (BCT)'s route 109 southbound bus line, "LAX City Bus Center - Redondo Beach Riviera Village". The hotel is 2 minutes from the Beryl 
-        St. / Harbor Dr. stop. It is also accessible via Metro's route 232 southbound bus line, "LAX City Bus Center - Long Beach".</p>
+          <div>
+            <sup>†</sup>Taxes and fees not included in price.
+          </div>
 
-      <p>After arriving at LAX, look for the pink LAX Shuttle sign on the Lower/Arrivals Level (regardless of terminal) and board the  Metro Connector Shuttle, taking you to the LAX/Metro 
```

Comment:
It looks like the paragraphs between these sections were entirely lost, which makes the content very hard to read. We'll need to find a way to fix this.